core/controlplane/config/templates/cloud-config-controller

```diff
@@ -1045,6 +1045,28 @@ write_files:
             initialDelaySeconds: 15
             timeoutSeconds: 15
 
+  - path: /etc/kubernetes/manifests/kube-rescheduler.yaml
+    content: |
+      apiVersion: v1
+      kind: Pod
+      metadata:
+        name: rescheduler-v0.2.2
+        namespace: kube-system
+        labels:
+          k8s-app: rescheduler
+          version: v0.2.2
+          kubernetes.io/cluster-service: "true"
+          kubernetes.io/name: "Rescheduler"
+      spec:
+        hostNetwork: true
+        containers:
+        - name: rescheduler
+          image: gcr.io/google-containers/rescheduler:v0.2.2
+          # TODO: Make resource requirements depend on the size of the cluster
```

Comment:
@mumoshu I copied this from https://github.com/kubernetes/kubernetes/commit/c304fa1f06eb23bcdc66b3f110f496681bc80301, it's what the salt setup has in the main repo. I'd be happy to just remove it perhaps as none of the others have it.